sites/authentication/_base.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * Specific classes for authentication section which include Sign-on/Sign-off button and the Contextual Sign-in button.
+ */
+
+.btn-signin button, .btn-signin a  {
```

Comment:
Considering those button are only and specifically styled for the authentication button, don't create new additional button style

You can use something relative to the id "wb-so" like:
```suggestion
# wb-so .btn  {
```